Automattic/jetpack
# AI Assistant: Update Breve implementation to React

Fixes #38124

## Proposed changes:

* Updates the implementation to the latest React one
* Fixes click-through issues

This PR is aimed at bringing parity with the main Breve implementation, not refactoring it. Subsequent PRs will start the divergence.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Enable the Breve feature flag
* On a post with some content, go to the Jetpack AI sidebar
* Open the AI Assistant panel
* Check that Breve works as it does on https://github.com/Automattic/Breve/pull/15
* Check that there are no console errors and clicking on highlighted text always works as expected.
